### PR TITLE
chore(release): prepare for release

### PR DESCRIPTION
**Pull request by Lyokone**
## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Review by rrousselGit — APPROVED** [review]

File: packages/firebase_auth/firebase_auth/pubspec.yaml [hunk 1]
```diff
@@ -15,8 +15,8 @@ environment:
 
 dependencies:
   firebase_auth_platform_interface: ^6.11.8
```
Comment (russellwheatley) [inline packages/firebase_auth/firebase_auth/pubspec.yaml:17]: This hasn't been updated.

File: packages/firebase_in_app_messaging/firebase_in_app_messaging/example/pubspec.yaml [hunk 2]
```diff
@@ -8,9 +8,9 @@ environment:
 dependencies:
   cupertino_icons: ^1.0.2
   firebase_analytics: ^10.1.1
```
Comment (russellwheatley) [inline packages/firebase_in_app_messaging/firebase_in_app_messaging/example/pubspec.yaml:10]: https://github.com/firebase/flutterfire/pull/10472/files#diff-21ed3552c47812b496298fbcdd3b59e7c913f057f4010af868b72104c299a205R7 This should be updated.

File: packages/firebase_messaging/firebase_messaging/example/pubspec.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ environment:
   flutter: '>=1.12.13+hotfix.5'
 
 dependencies:
-  firebase_core: ^2.5.0
+  firebase_core: ^2.6.0
   firebase_messaging: ^14.2.2
```
Comment (russellwheatley) [inline packages/firebase_messaging/firebase_messaging/example/pubspec.yaml:10]: https://github.com/firebase/flutterfire/pull/10472/files#diff-4d9a8918a1b3e8d9549bae0e9973c779ca6bd2b771b1ebb7f9791c2d761d4b70R6 should be updated.

File: packages/firebase_ui_database/example/pubspec.yaml [hunk 4]
```diff
@@ -29,11 +29,11 @@ environment:
...
   flutter:
     sdk: flutter
-  firebase_ui_database: ^1.0.13
+  firebase_ui_database: ^1.0.14
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
-  firebase_core: ^2.5.0
+  firebase_core: ^2.6.0
   firebase_database: ^10.0.10
```
Comment (russellwheatley) [inline packages/firebase_ui_database/example/pubspec.yaml:37]: https://github.com/firebase/flutterfire/pull/10472/files#diff-caca17354dc9cace561a8057a675e3be996a0deb3040b2405c11074176cb7384R6

File: tests/pubspec.yaml [hunk 5]
```diff
@@ -8,50 +8,49 @@ environment:
...
+  firebase_auth_web: ^5.2.6
+  firebase_core: ^2.6.0
   firebase_core_platform_interface: ^4.5.3
-  firebase_core_web: ^2.1.1
-  firebase_crashlytics: ^3.0.12
-  firebase_crashlytics_platform_interface: ^3.3.12
+  firebase_core_web: ^2.2.0
+  firebase_crashlytics: ^3.0.13
+  firebase_crashlytics_platform_interface: ^3.3.13
   firebase_database: ^10.0.10
```
Comment (russellwheatley) [inline tests/pubspec.yaml:32]: should be updated.

File: tests/pubspec.yaml [hunk 6]
```diff
@@ -8,50 +8,49 @@ environment:
...
 
 dependencies:
-  cloud_functions: ^4.0.8
-  cloud_functions_platform_interface: ^5.1.27
-  cloud_functions_web: ^4.3.16
+  cloud_functions: ^4.0.9
+  cloud_functions_platform_interface: ^5.1.28
+  cloud_functions_web: ^4.3.17
   collection: ^1.15.0
   firebase_analytics: ^10.1.1
```
Comment (russellwheatley) [inline tests/pubspec.yaml:15]: should be updated.

File: tests/pubspec.yaml [hunk 7]
```diff
@@ -8,50 +8,49 @@ environment:
...
-  firebase_database_web: ^0.2.1+20
-  firebase_dynamic_links: ^5.0.12
-  firebase_dynamic_links_platform_interface: ^0.2.3+27
+  firebase_database_platform_interface: ^0.2.2+19
+  firebase_database_web: ^0.2.1+21
+  firebase_dynamic_links: ^5.0.13
+  firebase_dynamic_links_platform_interface: ^0.2.3+28
   #firebase_in_app_messaging: any
   #firebase_in_app_messaging_platform_interface: any
   firebase_messaging: ^14.2.2
```
Comment (russellwheatley) [inline tests/pubspec.yaml:39]: should be updated.

**Review by russellwheatley — CHANGES_REQUESTED** [review]

**Pull request merged**